erc20-holdings/src/lib.rs

```diff
@@ -46,6 +46,7 @@ fn map_block_to_erc20_contracts(
         if call.call_type == pbeth::v2::CallType::Create as i32 {
             // skipping contracts that are too short to be an erc20 token
             if code_len(call) < 150 {
+                // contract byte code < 150 bytes
```

Comment:
Not a fan of comment as it's too similar looking to the above. Think maybe it could be better to change the fn name from code_len to something like contract_bytecode_len() instead?

Reply:
agreed, fixed